index.bs

```diff
@@ -1266,7 +1289,7 @@ To <dfn>serialize as a mapping</dfn> given |iterable|, |max depth|,
 Modules {#modules}
 ==================
 
-## Session ## {#module-session}
+## session ## {#module-session}
```

Comment:
```suggestion
## The session Module ## {#module-session}
```

I'm suggesting this in the common style of "The FooBar Interface", mostly to avoid having a lowercase section title.